### PR TITLE
[FML-35] - partially finish implementation.

### PR DESCRIPTION
**Pull request by dsuhinin**
┆Issue is synchronized with this [Jira Task](https://gr-oss.atlassian.net/browse/FML-45) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by suprjinx — DISMISSED** [review]
Looking good!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,6 +37,12 @@ jobs:
           mkdir pkg/ui/{aim,mlflow}/embed/build
           touch pkg/ui/{aim,mlflow}/embed/build/index.html
 
+      - name: Download mockery
+        run: go install github.com/vektra/mockery/v2@v2.20.0
+
+      - name: Generate mocks
+        run: make mocks-generate; sleep 5;
```
Comment (jgiannuzzi) [inline .github/workflows/ci.yml:44]: Why is there a `sleep 5` in there?

Comment (dsuhinin) [reply .github/workflows/ci.yml:44]: it could be a small lag, so mocks appear with a small delay. 5 seconds enough for all files to appear.

Comment (dsuhinin) [reply .github/workflows/ci.yml:44]: so just to make sure.

Comment (jgiannuzzi) [reply .github/workflows/ci.yml:44]: do you mean that when `make mocks-generate` is done, it's actually not fully done? 🤔

Comment (dsuhinin) [reply .github/workflows/ci.yml]: it seems like it works well without sleep but I had such a problem on my local. ok, then lets see.

**Review by fabiovincenzi — APPROVED** [review]

**Review by suprjinx — APPROVED** [review]
LGTM

File: pkg/api/mlflow/dao/convertors/log.go [hunk 2]
```diff
@@ -37,5 +41,33 @@ func ConvertLogBatchRequestToDBModel(
...
+	for n, metric := range req.Metrics {
+		m := models.Metric{
+			Key:       metric.Key,
+			Timestamp: metric.Timestamp,
+			Step:      metric.Step,
+			RunID:     runID,
+		}
+		if v, ok := metric.Value.(float64); ok {
+			m.Value = v
+		} else if v, ok := metric.Value.(string); ok {
```
Comment (dave-gantenbein) [inline pkg/api/mlflow/dao/convertors/log.go:55]: It would be great to have tests asserting these corner cases, and the invalid default case.
